smartcontractkit/external-adapters-js
# Only return string in WS deserialization if it's not JSON [develop]

Story: https://app.shortcut.com/chainlinklabs/story/23381/ws-string-deserializer-breaks-ws

## Changes

- The `deserialize()` function for WS will now only return a string if the payload cannot be parsed to JSON
- Make the debug log only happen if the message looked like JSON



## Steps to Test

1. Run 1forge WS
2. Run other WS adapter
3. Verify that both work with WS enabled (e.g. check the staleness value with `NODE_ENV=development`, verify you get `WS: Subscribed` and not just `WS: Connected`)

## Quality Assurance
